riotfile.py

```diff
@@ -203,6 +203,30 @@ def select_pys(min_version=min(SUPPORTED_PYTHON_VERSIONS), max_version=max(SUPPO
                 ),
             ],
         ),
+        Venv(
```

Comment:
is there a corresponding change to `tox.ini` needed?

Reply:
Missed that in the initial commit, updated!